bin/woff2_compress.js

```diff
@@ -18,22 +19,28 @@ var parser = new argparse.ArgumentParser({
 });
 
 parser.addArgument([ 'infile' ],  { nargs: 1, help: 'Input file' });
-parser.addArgument([ 'outfile' ], { nargs: 1, help: 'Output file' });
+parser.addArgument([ 'outfile' ], { nargs: '*', help: 'Output file (- for stdout)' });
 
 ////////////////////////////////////////////////////////////////////////////////
 
 let args = parser.parseArgs();
+const [ infile ] = args.infile;
+const [ outfile = infile.replace(extname(infile), '.woff2') ] = args.outfile;
```

Comment:
That breaks my brain.

Reply:
😅 he he.

Would you rather prefer to have the default value for the outfile being defined later or is it a matter of making the default assignment clearer?

I can also _not use_ destructuring and stay closer to the original codebase style.

Reply:
I'm ok with anything readable without calls to MDN and Stackoverflow :). No need to make difficulties for next potential maintainers.

Also:

- if file path has `.ttf` inside, your replace will do something wrong
- if extention is NOT `.ttf` - file will be rewritten.

Comments about extension are not critical, but worth to improve if you have a time.

Reply:
Node `path.extname` checks for the extension name only so it's safe even if `.ttf` appears in the middle of the path.

I'll make it clearer :-)